msm/src/serialization/witness.rs

```diff
@@ -210,13 +213,29 @@ impl<F: PrimeField> Env<F> {
             lookups,
         }
     }
+
+    // Commonly used by range checking functions.
+    fn record_range_check(
```

Comment:
This doesn't seem to be anything related to range checks. It is simply a lookup, right?